data/Taseski_2017/metadata.yml

```diff
@@ -91,7 +91,7 @@ traits:
   replicates: unknown
   methods: Following methods described in (Perez-Harguindeguy et al. 2013).
 - var_in: leaf_mass_per_area
-  unit_in: .na
+  unit_in: g/cm2
```

Comment:
Should we remove 'units unknown and nothing makes sense' from the description now?

Reply:
Good catch - I'll change